### DISCUSSION TITLE
Version 1.0.1

### DISCUSSION THREAD

**Discussion by bryceandy** [bryceandy/laravel-beem#1] [General]
This version adds support for PHP 8

<hr /><em>This discussion was created from the release <a href='https://github.com/bryceandy/laravel-beem/releases/tag/v1.0.1'>Version 1.0.1</a>.</em>

**Comment by Kivaru** [comment]
thank you for the package but am getting Undefined offset: 0 when I set no_redirection to false

**Reply by bryceandy** [reply]
There's no `no_redirection` on this package. Maybe you would want to use that on the [selcom](https://github.com/bryceandy/laravel-selcom) package.
